search-sentiments-map/public/scripts/map-script.js

```diff
@@ -66,17 +66,32 @@ function loadMapOutline() {
 
 /** Loads the country sentiment score from Datastore. */
 function loadCountryData() {
+  var dataVariableMax;
+  var countryMax = '';
+  var dataVariableMin; 
+  var countryMin = '';
   map.data.forEach(function (row) {
     const countryCode = row.getId();
+    const country = row.getProperty('name');
     let topicData = getCurrentCustomSearchData();
     let countryData = topicData.countries
       .filter(countries => countries.country === countryCode);
     let dataVariable = 0;
     if (countryData.length != 0) {
-      dataVariable = countryData[0].averageSentiment;
-      // TODO(ntarn): Remove console.log statements when finished debugging.
-      console.log('ntarn debug loadCountryData: ' + 'country' +
-          countryData[0].country + ' averageSentiment: ' + dataVariable);
+      let score = countryData[0].averageSentiment;
+      dataVariable = score;
+      if (dataVariableMax == undefined) {
+        dataVariableMax = score;
+        countryMax = country;
+        dataVariableMin = score;
+        countryMin = country;
```

Comment:
You can initialize dataVariableMax = Number.Min (or some small enough number) and dataVariableMin to be large enough so that you can get rid of this if part